agent/http.go

```diff
@@ -367,6 +367,9 @@ func (s *HTTPHandlers) wrap(handler endpoint, methods []string) http.HandlerFunc
 					continue
 				}
 				logURL = strings.Replace(logURL, token, "<hidden>", -1)
+				httpLogger.Warn("This request is using a token query parameter "+
+					"which is deprecated and will be removed in future Consul versions.",
```

Comment:
If we change the comment below, we might also say "will be removed in Consul 1.17" here.